apps/designer-standalone/src/app/AzureLogicAppsDesigner/DesignerCommandBar.tsx

```diff
@@ -53,9 +55,26 @@ export const DesignerCommandBar = ({
       skipValidation: false,
       ignoreNonCriticalErrors: true,
     });
-    await saveWorkflow(serializedWorkflow);
 
-    updateCallbackUrl(designerState, DesignerStore.dispatch);
+    const validationErrorsList = Object.entries(designerState.operations.inputParameters).reduce((acc, [id, nodeInputs]) => {
+      const hasValidationErrors = Object.values(nodeInputs.parameterGroups).every((parameterGroup) => {
+        return parameterGroup.parameters.every((parameter) => {
+          const validationErrors = validateParameter(parameter, parameter.value);
+          if (validationErrors.length > 0) {
+            dispatch(updateParameterValidation({ nodeId: id, groupId: parameterGroup.id, parameterId: parameter.id, validationErrors }));
+          }
+          return validationErrors.length;
+        });
+      });
+      return hasValidationErrors ? { ...acc, [id]: hasValidationErrors } : { ...acc };
+    }, {});
+
+    const hasEmptyRequiredInputs = !isNullOrEmpty(validationErrorsList);
```

Comment:
We might want to rename this, it's not always empty for errors, could be a wrong type or something

Reply:
ohhh yeaah, thats true.  I will rename it. Thanks